Cropper/package.json

```diff
@@ -1,53 +1,53 @@
 {
-  "name": "cropper",
-  "version": "4.0.2",
-  "description": "Image Cropping Widget",
-  "main": "index.js",
-  "scripts": {
-    "dev": "cross-env MODE=development webpack --config conf/webpack.config.js",
-    "build": "cross-env MODE=production webpack --config conf/webpack.config.js"
-  },
-  "keywords": [
-    "Mendix",
-    "widget",
-    "Dojo",
-    "Javascript",
-    "ES6",
-    "JQUERY"
-  ],
-  "author": {
-    "name": "Osama Najjar",
-    "email": "osama.najjar@mendix.com"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/preset-env": "^7.2.3",
-    "autoprefixer": "^9.4.3",
-    "babel": "^6.23.0",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-add-module-exports": "^1.0.0",
-    "concurrently": "^4.1.0",
-    "cross-env": "^5.2.0",
-    "css-loader": "^2.1.0",
-    "cssnano": "^4.1.8",
-    "file-loader": "^3.0.1",
-    "fs-extra": "^7.0.1",
-    "imports-loader": "^0.8.0",
-    "mini-css-extract-plugin": "^0.5.0",
-    "node-sass": "^4.13.0",
-    "postcss-clean": "^1.1.0",
-    "postcss-loader": "^3.0.0",
-    "sass-loader": "^7.2.0",
-    "style-loader": "^0.23.1",
-    "uglifyjs-webpack-plugin": "^2.1.1",
-    "webpack": "^4.28.2",
-    "webpack-archive-plugin": "^3.0.0",
-    "webpack-cli": "^3.1.2",
-    "xml-webpack-plugin": "0.0.2"
-  },
-  "dependencies": {
-    "jquery": "^3.5.0",
-    "jquery-jcrop": "^0.9.13"
-  }
+    "name": "cropper",
+    "version": "4.0.6",
+    "description": "Image Cropping Widget",
+    "main": "index.js",
+    "scripts": {
+        "dev": "cross-env MODE=development webpack --config conf/webpack.config.js",
+        "build": "cross-env MODE=production webpack --config conf/webpack.config.js"
+    },
+    "keywords": [
+        "Mendix",
+        "widget",
+        "Dojo",
+        "Javascript",
+        "ES6",
+        "JQUERY"
+    ],
+    "author": {
+        "name": "Osama Najjar",
+        "email": "osama.najjar@mendix.com"
+    },
+    "license": "MIT",
```

Comment:
Should we update the license here as well?

And should we add repository field to avoid the npm warning of this missing field?

Reply:
> Should we update the license here as well?

Let's keep it MIT because of export control. Also, module itself has Apache-2.0

Reply:
Repository url added